app/sessionStore.js

```diff
@@ -412,6 +412,7 @@ module.exports.defaultAppState = () => {
         guid: [],
         timestamp: 0
       }
-    }
+    },
+    menubar: {}
```

Comment:
will keep it here after talking w/ @bridiver; in the future, we can introduce more window specific state in the appState (deprecating windowStore)